pkg/trace/info/stats.go

```diff
@@ -114,10 +119,18 @@ func newTagStats(tags Tags) *TagStats {
 	return &TagStats{tags, Stats{}}
 }
 
+// AllNormalizationIssues returns a map of counts of all normalization issue reasons due to dropped or malformed traces
```

Comment:
Let's remove this. It's not used anymore.

Reply:
Removed